node-require/add.js

```diff
@@ -0,0 +1,4 @@
+function add(x, y) {
+  return x + y;
+}
+module.exports = add(Number(process.argv[2]), Number(process.argv[4]));
```

Comment:
This isn't how we'd normally export something. Usually you just export the function, by name:
`modules.exports = add`